internal/pldag/model.go

```
@@ -25,6 +25,10 @@ func New() *Model {
 func (m *Model) SetPrimitives(primitives ...string) {
 	deduped := utils.Dedupe(primitives)
 	for _, d := range deduped {
+		if d == "" {
```

Personally I prefer throwing an error instead of ignoring the empty string.

---

Agree, I will change change it